e2e-tests/playwright/tests/functional/system_console/mobile_security.spec.ts

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {expect, test} from '@e2e-support/test_fixture';
+
+test('should be able to enable mobile security settings when licensed', async ({pw}) => {
+    const {adminUser, adminClient} = await pw.initSetup();
+
+    const license = await adminClient.getClientLicenseOld();
+
+    test.skip(license.SkuShortName !== 'enterprise', 'Skipping test - server has no enterprise license');
```

Comment:
@yasserfaraazkhan @larkox  Where are the licenses set up for these E2E runs? If the license is not setup properly then skipping defeats the purpose of the test?

Reply:
That is a great question. Maybe @yasserfaraazkhan has more insights on this.

My approach has been: Create a test for whenever there is a license, and another tests when there is no llicense, to have all cases covered.

Not sure if we are using both kind of environments for tests in any scenario, but sounds like something we should do.